src/app/views/company/company.controller.coffee

```diff
@@ -17,7 +17,8 @@ angular.module 'mnoEnterpriseAngular'
         if vm.isBillingShown()
           vm.activeTab = 'billing'
```

Comment:
Can this be moved into the `setActiveTab()` method to keep the logic all together?

Reply:
Agreed